src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -36,13 +36,19 @@ public EventDTO createEvent(EventDTO eventDTO) {
     }
 
     @Override
-    public List<EventDTO> deleteEvent() {
-        return null;
+    public void  deleteEvent(Long id) {
+    Event event = eventRepository.findById(id).orElseThrow(()-> new EventNotFoundException("Event not found with id: " + id ));
+    eventRepository.deleteById(id);
```

Comment:
Please consider formatting this class and using a logger after each repository action.